<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QDialog" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartRadio - Create Account</string>
  </property>
  <widget class="QWidget" name="mainContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="welcomeTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>join smart radio</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeSubtitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CREATE YOUR STUDIO IDENTITY</string>
    </property>
   </widget>
   <widget class="QFrame" name="signupCard">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>340</width>
      <height>500</height>
     </rect>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string>SR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>choose username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>email address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>choose password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>260</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>register now</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>415</y>
       <width>260</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>back to login</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
